staking/reward/values.go

```diff
@@ -0,0 +1,155 @@
+package reward
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/harmony-one/harmony/common/denominations"
+	"github.com/harmony-one/harmony/consensus/engine"
+	shardingconfig "github.com/harmony-one/harmony/internal/configs/sharding"
+	"github.com/harmony-one/harmony/internal/params"
+	"github.com/harmony-one/harmony/numeric"
+	"github.com/harmony-one/harmony/shard"
+)
+
+var (
+	// PreStakedBlocks is the block reward, to be split evenly among block signers in pre-staking era.
+	// 24 ONE per block
+	PreStakedBlocks = new(big.Int).Mul(big.NewInt(24), big.NewInt(denominations.One))
```

Comment:
PreStakedBlockReward?   same for others.

Reply:
I purposefully removed the 'reward' part because of how these values are used. It's redundant to add 'reward' at the end since all the values are in the reward pkg.

For example `reward.StakedBlocks` v.s. `reward.StakedBlocksReward`. The former flows better and is easier to read.

Reply:
ok cool.